Stratis.Bitcoin/Features/Consensus/CoinViews/DBreezeCoinView.cs

```diff
@@ -6,239 +6,302 @@
 using NBitcoin.BitcoinCore;
 using Stratis.Bitcoin.Configuration;
 using Stratis.Bitcoin.Utilities;
+using DBreeze.DataTypes;
 
 namespace Stratis.Bitcoin.Features.Consensus.CoinViews
 {
-	public class DBreezeCoinView : CoinView, IDisposable
-	{
-		private readonly DBreezeSingleThreadSession session;
-		private readonly Network network;
-		private uint256 blockHash;
-		private readonly BackendPerformanceCounter performanceCounter;
-
-		public DBreezeCoinView(Network network, DataFolder dataFolder) 
-			: this(network, dataFolder.CoinViewPath)
-		{
-		}
-
-		public DBreezeCoinView(Network network, string folder)
-		{
-			Guard.NotNull(network, nameof(network));
-			Guard.NotEmpty(folder, nameof(folder));
+    /// <summary>
+    /// Persistent implementation of coinview using DBreeze database.
+    /// </summary>
+    public class DBreezeCoinView : CoinView, IDisposable
+    {
+        /// <summary>Database key under which the block hash of the coin view's current tip is stored.</summary>
+        private static readonly byte[] blockHashKey = new byte[0];
+
+        /// TODO: Can we removed this? It is not used anywhere.
```

Comment:
yes

Reply:
yes

Reply:
going to the new issue as you merged this one